frontend/src/lib/auth/index.ts

```diff
@@ -86,3 +87,95 @@ export async function getRequestToken() {
     secret: params.get("oauth_token_secret"),
   };
 }
+
+export const auth = async ({
+  token,
+  secret,
+}: {
+  token?: string | null;
+  secret?: string | null;
+}) => {
+  const cookies = await cookiesPromise();
+  const accessToken = token ?? cookies.get("access_token")?.value;
+  const accessSecret = secret ?? cookies.get("access_token_secret")?.value;
+
+  if (accessToken === "" || accessSecret === "") {
+    return false;
+  }
+
+  try {
+    const response = await fetch("https://planer.solvro.pl/api/v1/user/login", {
```

Comment:
czemu ten adres jest na sztywno wpisany

Reply:
a czm nie, pozniej bede sie bawic w zmienne

Reply:
no o to chodzi, żebyś nie musiał mergować zmian na maina, żeby testować backend 😭

tylko odpalasz sobie oba lokalnie i lecisz